.github/workflows/boards_build.yml

```diff
@@ -0,0 +1,38 @@
+---
+name: Build supported boards
+on:
+  push:
+    paths:
+      - 'patches/tock/**'
```

Comment:
We don't have subdirectory in `patches/` so you can remove a star in the pattern here.

Reply:
So what are the rules for these patterns?

I'm also afraid that by over-optimizing the patterns, we may miss regressions because the wrong rule prevented the right test to run.

Except for the crypto tests, all the workflows run in a reasonable time (and GitHub is faster than Travis in that regard), so we should rather remove most of the rules to focus on a few rules for the cases where it's *really* slow, than having complex rules that let regressions through.